test/connectivity/README.md

```diff
@@ -1,3 +1,58 @@
-# Connectivity tests
+ Connectivity tests
 
+![Connectivity Test Diagram](./images/ConnectivityTestDiagram.png)
 
+### Test Scenarios
+These are test scenarios to be covered in Connectivity test
+
+1. Messaging Test
+   1. Module-to-Module
+   2. Module-to-IoT Hub
+2. Twin Property Update Test
+   1. Desired property
+   2. Reported property
+3. Direct Method Test
+   1. Cloud-to-module
+   2. Ping method in EdgeAgent
+4. EdgeAgent Deployment Test
+   1. Deployment change
+
+### General flow
+1. Currently Connectivity test is supported only to run on Linux AMD64.
+2. Connectivity test can be run by calling [Connectivity Test script](https://github.com/Azure/iotedge/blob/master/test/connectivity/scripts/connectivityTest.sh).
+3. Each test will be run using AMQP and MQTT TCP protocol.
+4. For each run, a new tracking id (Guid) will be generated by the test script.
+5. Test will be started after pre-defined warm-up period (default 2 minutes), which assume EdgeHub and other modules containers are up and running.
+6. A test duration is configurable and is set to 1 hour by default.
+7. A test module is responsible for providing the Test Result Coordinator with a result of its operation.
+8. An operation result is persistently stored by the Test Result Coordinator after a test validation is completed.
```

Comment:
operation result is stored when received by Test Result Coordinator, not after test validation is completed

Reply:
The validation is to ensure type property of TestOperationResult can cast to TestOperationResultType.